specs/waku/v2/waku-filter.md

```diff
@@ -116,8 +126,9 @@ implementation, though a reasonable default is one minute.
 ### Next version
 
 - Fix: Ensure contentFilter and contentTopic are repeated fields, per implementation
+- Change: Add ability to unsubscribe from filters. Make `subscribe` an explicit boolean indication.
 
-### 2.0.0-beta1
+### 2.0.0-beta2
```

Comment:
Do we want to bump in this PR or later? If this PR, add a new section for beta2 and bump header. If later, don't change this line

Reply:
I think we want to bump in this PR (also given comments below). Can the beta2 section then be left open for now since there is no matching implementation?

Reply:
Sounds good, I think filling it out now is fine.

Strictly speaking, nim-waku should state in docs that it supports beta1 version, and the unsubscribe method here might be marked as "experimental" since there is no implementation yet. But since you are going to push that soon presumably I'd just mark it now.